tests/test-pkcs11-tool-test-threads.sh

```diff
@@ -14,16 +14,16 @@ fi
 card_setup
 
 echo "======================================================="
-echo "Test pkcs11 threads OSILGISLT0 "
+echo "Test pkcs11 threads IN "
 echo "======================================================="
-$PKCS11_TOOL --test-threads IN -L
+$PKCS11_TOOL --test-threads IN -L --module="$P11LIB"
 assert $? "Failed running tests"
 
 
 echo "======================================================="
 echo "Test pkcs11 threads OSILGISLT0 "
 echo "======================================================="
-$PKCS11_TOOL --test-threads OSILGISLT0 -L
+$PKCS11_TOOL --test-threads OSILGISLT0 -L --module="$P11LIB"
```

Comment:
@dengert The `pkcs11-tool --test-threads` here warns about unknown operation `OS`. Can you check what you had in your mind when writing this test?

Reply:
It looks like OS was the old way of initialize with OS locking. But the code was change to use:
- IN call C_Initialize with NULL 
- IL  call C_Initialize with CKF_OS_LOCKING_OK

The OS was left in the test script input parameter and comment . It should look like this: 
```
./pkcs11-tool --test-threads ILGISLT0 -L
Test thread 0 started with options:ILGISLT0
Test thread 0 C_Initialize CKF_OS_LOCKING_OK 
Main C_Initialize(NULL) rv:CKR_OK
Test thread 0 C_Initialize  returned CKR_CRYPTOKI_ALREADY_INITIALIZED
Test thread 0 C_GetInfo
Test thread 0 C_GetInfo returned CKR_OK
Test thread 0 C_GetSlotList to get l_p11_num_slots
Available slots:
No slots.
test_threads cleanup starting
test_threads threads stills active:1
test_threads thread:0 state:2, rv:CKR_OK

test_threads waiting for 30 seconds ...
Test thread 0 C_GetSlotList returned CKR_OK
Test thread 0 l_p11_num_slots:0
Test thread 0 C_GetTokenInfo from slot_index 0 using show_token
Segmentation fault (core dumped)
```

But without a reader and card, there was a segfault that I will correct.
Without a slot, the test code called `show_token(p11_slots[(*(pctest + 1) - '0')]);` line 7512 should be an error message.

I must not have tested without a reader.

Reply:
Thanks. Let me know if you would like to submit it as a separate PR there will be some commit I should pull.

Reply:
Submitted #2388 to fix both.